maven_plugin_config-master/pom.xml

```
@@ -24,7 +24,7 @@
   <description>Maven Master Plugin Build Configuration - General</description>
 
   <properties>
-    <jdk.min.version>25</jdk.min.version>
+    <jdk.min.version>24</jdk.min.version>
```

Just as reminder: revert this to Java 25

```suggestion
    <jdk.min.version>25</jdk.min.version>
```